studio-docs/source/setup-analytics.mdx

```diff
@@ -55,21 +55,21 @@ You can set up a reporting agent in your GraphQL server to push metrics to Apoll
 * Emitting batches of traces to the Apollo Studio reporting endpoint
 * Optionally defining plugins to enable advanced reporting features
 
-Apollo Server defines its agent for performing these tasks in [`agent.ts`](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting/src/agent.ts).
+Apollo Server defines its agent for performing these tasks in [the usage reporting plugin](https://github.com/apollographql/apollo-server/tree/main/packages/apollo-server-core/src/plugin/usageReporting).
 
 > If you're interested in collaborating with Apollo on creating a dedicated integration for your GraphQL server, please get in touch with us at **support@apollographql.com** or via our [Apollo Spectrum Community](https://spectrum.chat/apollo).
 
 ### Tracing format
 
 Apollo Studio's reporting endpoint accepts batches of traces that are encoded in **protocol buffer** format. Each trace corresponds to the execution of a single GraphQL operation, including a breakdown of the timing and error information for each field that's resolved as part of the operation.
 
-The schema for this protocol buffer is defined as the `FullTracesReport` message in the [TypeScript reference implementation](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting-protobuf/src/reports.proto#L466). 
+The schema for this protocol buffer is defined as the `Report` message in the [TypeScript reference implementation](https://github.com/apollographql/apollo-server/blob/main/packages/apollo-reporting-protobuf/src/reports.proto#L350).
```

Comment:
thanks for tracking down the line! Should we consider just dropping the line reference here altogether, since this is bound to change again?

Reply:
Yeah it probably isn't that helpful, I dunno.

Reply:
Removing line from the link.